src/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+                    <path className="transition-colors ease-in delay-100 group-hover:fill-[#F7613E]" d="M40.9752 36.2486C40.691 35.9371 40.2885 35.6265 39.6537 35.2425C39.5621 35.1914 39.4548 35.1232 39.3557 35.0638C38.8271 34.7459 38.1903 34.4114 37.4466 34.0076C36.304 33.3803 35.1606 32.7688 34.0065 32.1623L31.9673 31.0808C31.8616 31.0498 31.7543 30.9734 31.6546 30.9254C31.2629 30.7381 30.8474 30.5632 30.4164 30.483C30.2677 30.4545 30.117 30.4389 29.9659 30.4362C29.8681 30.435 29.7708 30.441 29.674 30.4539C29.2663 30.5171 28.8888 30.7074 28.5954 30.9979C28.302 31.2883 28.1077 31.6638 28.04 32.0711C28.009 32.3302 28.0184 32.5924 28.0682 32.8486C28.1678 33.3913 28.4101 33.928 28.6608 34.3981L29.7498 36.4391C30.3557 37.5902 30.9689 38.7347 31.5977 39.8755C32.0031 40.6196 32.3407 41.2566 32.6568 41.7844C32.7168 41.8837 32.7847 41.9905 32.8359 42.0824C33.2207 42.7171 33.5299 43.1165 33.8434 43.403C34.0464 43.599 34.2899 43.7482 34.5565 43.8404C34.8368 43.9333 35.1336 43.9663 35.4274 43.9372C35.577 43.9194 35.7255 43.8938 35.8722 43.86C36.0168 43.8213 36.1592 43.775 36.2988 43.7212C36.4497 43.6646 36.5962 43.5969 36.7372 43.5188C37.5651 43.0546 38.3282 42.4832 39.0069 41.8195C39.8208 41.0185 40.5403 40.1452 41.0999 39.1453C41.1778 39.0038 41.2451 38.8569 41.3016 38.7058C41.3544 38.5658 41.4001 38.4233 41.4384 38.2788C41.4714 38.1319 41.4968 37.9832 41.5144 37.8338C41.5423 37.54 41.5082 37.2439 41.4143 36.9642C41.322 36.6964 41.172 36.452 40.9752 36.2486Z" fill="#FFFDF9"/>
+                </svg>
+            </a>
+        </div>
+
+        <div className="footerText md:order-3 md:basis-1/2 mb-3 mx-12 md:text-md">
+            <p className="text-2xl md:text-4xl md:pt-10 my-8">Poke Bay</p>
```

Comment:
I think I changed this recently so that's my bad but please increase the font weight